flink-metrics/flink-metrics-jmx/src/test/java/org/apache/flink/metrics/jmx/JMXReporterTest.java

```diff
@@ -297,4 +342,42 @@ public long getMin() {
 			};
 		}
 	}
+
+	static class TestingMeter implements Meter {
```

Comment:
let's put this in a separate utility test class under `flink-runtime/src/test/java/org/apache/flink/runtime/metrics/util`